prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/translations/README.txt

```diff
@@ -0,0 +1,4 @@
+This translations directory is for .po|.mo files
+
+Visit XBlock Tutorial docs on Internationalization :
+https://github.com/edx/edx-documentation/blob/master/en_us/xblock-tutorial/source/edx_platform/edx_lms.rst
```

Comment:
I updated the copy to be a little more clear, and changed the reference to point to the actual RTD location that will be updated by edx/edx-documentation#904